pykafka/cli/kafka_tools.py

```diff
@@ -54,12 +55,23 @@ def fetch_consumer_lag(client, topic, consumer_group):
     """
     latest_offsets = fetch_offsets(client, topic, 'latest')
     consumer = topic.get_simple_consumer(consumer_group=consumer_group,
-                                         auto_start=False, 
+                                         auto_start=False,
                                          reset_offset_on_fetch=False)
     current_offsets = consumer.fetch_offsets()
-    return {p_id: (latest_offsets[p_id].offset[0], res.offset)
-            for p_id, res in current_offsets}
-
+    pid_dict = {}
+    for p_id, stat in current_offsets:
+        # Check if metadata stored into current_offsets
+        consumer_id = None
+        hostname = None
+        if bool(stat.metadata):
```

Comment:
This `bool` cast is unnecessary since the `if` operator casts its argument to a boolean.

Reply:
Fixed